doc/release-notes/0.37/0.37.md

```diff
@@ -28,7 +28,7 @@ These release notes support the [Eclipse OpenJ9&trade; 0.37.0 release plan](http
 
 ## Supported environments
 
-OpenJ9 release 0.37.0 supports OpenJDK 19.
+OpenJ9 release 0.37.0 supports OpenJDK 19. OpenJDK 19 is out of support at the time of the 0.37 release. Builds of 0.37 should not be used in production and may contain known security vulnerabilities as of April 18, 2023.
```

Comment:
Since OpenJ9 release 0.37.0 does not support OpenJDK 19, retaining the first sentence will be confusing. Unless there is a reason for retaining that, the first sentence should be deleted.

In the other 2 sentences, change "0.37 release" to "0.37.0 release"

Change "may" to "might".

Change the date to "18 April 2023" (as per the IBM style)

Reply:
I think we should keep sometime like "OpenJ9 release 0.37.0 supports OpenJDK 19.", but I've changed "supports" to "works with" so it's less confusing. I think I've make all the other changes requested here and in the next comment.